cmake/iwyu.cmake

```diff
@@ -2,6 +2,6 @@ find_program(iwyu_path NAMES include-what-you-use iwyu)
 if(iwyu_path)
   set_property(TARGET ${lib}
     PROPERTY ${iwyu_path} -Xiwyu
-      --mapping_file ${CMAKE_CURRENT_SOURCE_DIR}/iwyu.impl
+      --mapping_file ${CMAKE_CURRENT_SOURCE_DIR}/iwyu.imp
```

Comment:
https://github.com/ArthurSonzogni/FTXUI/blob/d9241435ce9fa3d81f0ea5a0d34c3048883629ee/iwyu.imp

Reply:
Thanks! I will take a look tomorrow!
This looks good. I only need to check why this wasn't an issue for me.